docs/content_management/create_edit_content_items.md

```diff
@@ -184,6 +186,32 @@ add or delete rows and columns in any part of the table, as well as merge and sp
 Your installation can also have custom elements available in the Rich Text field.
 Contact your website administrator about the details of using them.
 
+### AI Assistant
+
+If the [AI Actions](ai_actions.md) LTS update has been installed and configured in your application, when writing content, the **Improve text** button is visible in the toolbox.
+If you have `Action configuration/View` and `Action configuration/Execute` permissions, you can request that the AI Assistant performs specific tasks with your text.
```

Comment:
I'd explain that with the "Refine text" action type. Something like "the refine text actions that are defined will be offered as a choice".